src/containers/network/AboutNetwork.js

```diff
@@ -1,3 +1,17 @@
 import React from 'react'
-const AboutNetwork = props => <span />
-export default AboutNetwork
+import { connect } from 'react-redux'
+import { markdown } from '../../util/text'
+const { object } = React.PropTypes
+
+@connect((state, { params }) => ({network: state.networks[params.id]}))
+export default class AboutNetworks extends React.Component {
```

Comment:
see if you can stick with stateless components unless absolutely necessary. they're more performant (according to the React docs) and help us avoid antipatterns. 

decorators just use function composition, so you can do:

```js
const MyComponent = connect(...)(props => { ... })
```
or
```js
import { compose } from 'redux'
const MyComponent = compose(
  prefetch(...),
  connect(...)
)(props => { ... })
```

you can declare propTypes thus:
```js
MyComponent.propTypes = { ... }
```